changelog.txt

```diff
@@ -1,10 +1,19 @@
 *** Changelog ***
 
-= 4.5.4 - 2020-xx-xx =
-* Add   - Security.md with security and vulnerability reporting guidelines.
+= 4.5.4 - 2020-11-16 =
+* Add   - Stripe Connect OAuth.
 * Tweak - Add site_url to all transactions, not just recurring ones.
 * Add   - Customer's full name is now included in Stripe Customer object if available.
 * Add   - Accept payments via AliPay when store currency is set to CNY.
+* Tweak - Wording of Apple Pay link.
+* Fix   - Serve Apple Pay domain verification file via a rewrite rule.
+* Add   - Apple Pay holiday marketing notice.
```

Comment:
Should this be expanded to include https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1337?

Reply:
Thank you - yes

Reply:
Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1370/commits/e9c0beb1279601be30bed09de9363ef92a4a7853